pkg/rtc/mediatrack.go

```diff
@@ -430,3 +433,9 @@ func (t *MediaTrack) SetMuted(muted bool) {
 
 	t.MediaTrackReceiver.SetMuted(muted)
 }
+
+func (t *MediaTrack) handleReceiverEverAddDowntrack() {
+	if !t.everSubscribed.Swap(true) && t.params.OnTrackEverSubscribed != nil {
+		go t.params.OnTrackEverSubscribed(t.ID())
+	}
+}
```

Comment:
Wondering if this can be put in `MediaTrackReceiver` which can potentially handle remote tracks also. But, that will make harder to use the opts bit for creating receiver.

Reply:
the relaydowntrack will be added to webrtcreceiver so remote subscription will fire the event too

Reply:
Ah true, my bad.